portal-impl/src/com/liferay/portal/service/impl/LayoutLocalServiceHelper.java

```diff
@@ -713,6 +726,12 @@ protected boolean hasGuestViewPermission(Layout layout)
 	private static final Log _log = LogFactoryUtil.getLog(
 		LayoutLocalServiceHelper.class);
 
+	private static volatile LayoutFriendlyURLEntryValidator
+		_layoutFriendlyURLEntryValidator =
+			ServiceProxyFactory.newServiceTrackedInstance(
+				LayoutFriendlyURLEntryValidator.class,
+				LayoutLocalServiceHelper.class,
+				"_layoutFriendlyURLEntryValidator", false);
```

Comment:
Can you initialize the tracker indicating that it should use `null` instead of a proxy (there's an extra parameter that you need to provide for that). Otherwise, a proxy will be used, and the null check above will do nothing. Alternatively, keep this call as is, but removing the null check above.

Reply:
The parameter of the method is _useNullAsDummyService_. 
Is it right?

Reply:
Yes.

Reply:
@noornajj , can you update the PR with this changes?

Thanks!!

Reply:
Sure, I'll send a new PR